source/patterns/@aws-solutions-constructs/core/lib/apigateway-helper.ts

```diff
@@ -21,13 +21,14 @@ import * as apiDefaults from './apigateway-defaults';
 import { buildLogGroup } from './cloudwatch-log-group-helper';
 import { overrideProps, addCfnSuppressRules } from './utils';
 import { IRole } from '@aws-cdk/aws-iam';
+import { Construct } from '@aws-cdk/core';
```

Comment:
Didn't you want to delete line 17 along with adding this? Seems silly to import the reference in 2 ways.

Reply:
No, line 17 import is used in line 48

During the CDK v2 build, line 17 will be replaced from:

```
import * as cdk from '@aws-cdk/core';
```
to
```
import * as cdk from 'aws-cdk-lib';
```

And the reference to `cdk` from line 48 will work:

```
resources: [`arn:${cdk.Aws.PARTITION}:logs:${cdk.Aws.REGION}:${cdk.Aws.ACCOUNT_ID}:*`]
```